### DISCUSSION TITLE
Unable to verify school email

### DISCUSSION THREAD

**Discussion by UnicornRidingUnicorn** [community/community#139611] [GitHub Education]
### Hi everyone,

For some reason I'm not receiving the email to verify my school email address. I've checked junk and quarantine and I see nothing. Any thoughts?

**Comment by davevad93** [comment]
Hi @UnicornRidingUnicorn ,

Your academic institution may use a firewall that restricts incoming emails from domains outside your institution, in which case, you should ask your school's network administrator.

If that's not the problem, try contacting support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your issue.

**Reply by UnicornRidingUnicorn** [reply]
They're using O365 so who knows. I did open a ticket a few days ago. No response.

**Reply by davevad93** [reply]
I see, support can take a while to respond given the high number of tickets they have to deal with on a daily basis, I hope they can assist you as soon as possible.

**Reply by UnicornRidingUnicorn** [reply]
Thanks I appreciate it!
